test/system/030-run.bats

```diff
@@ -711,6 +711,18 @@ EOF
     run_podman rmi nomtab
 }
 
+@test "podman run --hostuser tests" {
+    skip_if_not_rootless "test whether hostuser is successfully added"
```

Comment:
why does it need rootless?

Reply:
Mainly because I am not sure how to figure out a user which is not in the container to add to the container.  If you are running rootless, I can add the current user.

If running as root, I could add a user to the system and then use that, but I am not crazy about modifying a system when users run the tests.

@edsantiago Any ideas?